src/cmap/connection_pool_events.ts

```diff
@@ -29,11 +29,18 @@ export class ConnectionPoolMonitoringEvent {
 export class ConnectionPoolCreatedEvent extends ConnectionPoolMonitoringEvent {
   /** The options used to create this connection pool */
   options?: ConnectionPoolOptions;
+  /** The size of the ConnectionPool's waitQueue */
+  waitQueueSize: number;
```

Comment:
This field is required by the [CMAP logging spec](https://github.com/mongodb/specifications/blob/master/source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#pool-created-message) if the driver supports the option and the user specifies it

Reply:
We have a `waitQueueSize` property which represents how large the wait queue currently is, but I think that "only required to include if the driver supports this option and the user specified a value." clarifies that it's only if it is a configurable option, which it is not IIUC.